### PR TITLE
Clarification of enable/disable mTLS

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

File: _docs/tasks/security/faq.md [hunk 1]
```diff
@@ -9,17 +9,20 @@ type: markdown
 ---
 {% include home.html %}
 
+* _How can I enable/disable mTLS encryption after I installed Istio?_
+
+  Enabling/disabling mTLS requires uninstalling and installing Istio,
+  as well as re-deploying your applications.
```
Comment (ldemailly) [inline _docs/tasks/security/faq.md]: you don't need to redeploy do you ?

Comment (ldemailly) [reply _docs/tasks/security/faq.md]: Let's put something like this as the answer:

```
kubectl edit configmap -n istio-system istio
```
comment out or uncomment out `authPolicy: MUTUAL_TLS`
and then
```
kubectl delete pods -n istio-system -l istio=pilot
```
to restart pilot, after a few seconds (depending on your `*RefreshDelay`) your envoy will have picked up the change

Comment (andraxylia) [reply _docs/tasks/security/faq.md]: I removed the requirement to re-deploy the apps, ptal.

Comment (ldemailly) [reply _docs/tasks/security/faq.md]: can you add the above in a "short cut" or "advanced users" section
(you asked me if I personally verified it works; I did spend the time to do so, it'd be nice to have it there)

Comment (andraxylia) [reply _docs/tasks/security/faq.md]: Agree, let me add it an a separate PR, since this one got merged.

Comment (ldemailly) [reply _docs/tasks/security/faq.md]: thanks!

**Pull request merged**